### PR TITLE
support mailgun reply_to

### PR DESCRIPTION
**Pull request by iCell**
Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes:

- [x] Run `bundle exec rspec` from the subdirectory of each tool you modified. Alternatively, run `rake test_all` from the root directory.
- [x] Run `bundle exec rubocop -a` to ensure the code style is valid
- [x] Read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] We currently don't accept new actions, please publish a plugin instead, more information in [Plugins.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Plugins.md)

Before submitting a pull request, we appreciate if you create an issue first to discuss the change :+1:

### CODE REVIEW

File: fastlane/lib/fastlane/actions/mailgun.rb [hunk 1]
```diff
@@ -82,6 +82,11 @@ def self.available_options
                                       description: "Mail HTML template",
                                       optional: true,
                                       is_string: true)
+          FastlaneCore::ConfigItem.new(key: :reply_to,
+                                      env_name: "MAILGUN_REPLAY_TO",
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/mailgun.rb]: I think this should be `MAILGUN_REPLY_TO`.

**Review by ohayon — CHANGES_REQUESTED** [review]
Hey @iCell, thanks for the PR! 

I just left one note about a typo in the `env_name` key of the `ConfigItem`.

Just to be safe, would you mind adding a test to `maligun_spec.rb` as well?

File: fastlane/lib/fastlane/actions/mailgun.rb [hunk 2]
```diff
@@ -103,6 +108,7 @@ def self.mailgunit(options)
         RestClient.post "https://api:#{options[:apikey]}@api.mailgun.net/v3/#{sandbox_domain}/messages",
                         from: "#{options[:from]}<#{options[:postmaster]}>",
                         to: (options[:to]).to_s,
+                        "h:Reply-To": options[:reply_to],
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/mailgun.rb]: What happens when the value is nil? does the API handle this properly? Maybe this value shouldn't be passed when it's not set by the user.

**Comment by KrauseFx** [comment]
Hey @iCell, have you had the time to look into the proposed changes and update the PR? 👍

**Comment by iCell** [comment]
@KrauseFx Sorry for not responding, I'm really busy with my recent work, I'll update the PR in a few days.

File: fastlane/lib/fastlane/actions/mailgun.rb (hunk 1 above)
Comment (iCell) [inline fastlane/lib/fastlane/actions/mailgun.rb]: @ohwutup  Sorry for the misspell

**Comment by iCell** [comment]
I have run the command `bundle exec rspec`, but it raised an exception `Fastlane Fastlane::FastFile Mailgun Action raises an error if no mailgun apikey is given`, is this exception normal？

**Comment by iCell** [comment]
I have run `bundle exec rspec` from the subdirectory of each tool I've modified, and the result is 0 failure. But through the results of Github CI, these errors are not caused by my code in the PR.

**Pull request closed without merging**